1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,12 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents = group.every(isIncluded ) // complete this statement
+   function isIncluded (group){
+        for (let i=0 ; i < group.length; i++){
+           return students.includes('group[i]');
```

Comment:
This would not pass when the group only has students. There are a couple of errors, first in the line `return students.includes('group[i]');` group should not be given as an array as group is the input to the function. But also, the input to the function here is each value inside of the array therefore you do not need a for loop in the function isIncluded.

One thing I wanted to mention. In this case, the test passes because the only example is false and the function return false, but it is important that you check that function also has the right functionality when you expect it to be true. For example, you can change the line 6 to `let group = ["Austine", "Dany", "Swathi"];` and check what happens then (it should be true but with your function it will still be false).

Reply:
Hi, Martin
You're absolutely right !
I had double checked it and re-wrote this function as following and it returns the opposite when  I deleted ''Daniel" from group array:
 **let groupIsOnlyStudents = group.every(x=>students.includes(x))** 
 
 Thanks!
 Howard

Reply:
Yes, that works!